src/plots/gl3d/scene.js

```diff
@@ -508,7 +508,7 @@ proto.plot = function(sceneData, fullLayout, layout) {
                 var obj = objects[j];
                 var objBounds = obj.bounds;
                 // add worse-case pad for cone traces
-                var pad = (obj._trace.data._normMax || 0) * dataScale[i];
+                var pad = 0.75 * (obj._trace.data._compMax || 0) * dataScale[i];
```

Comment:
This gives a better estimate. The data off https://github.com/plotly/plotly.js/pull/2641#issuecomment-390040970 now gives:

![image](https://user-images.githubusercontent.com/6675409/40211199-3f0a7b1c-5a17-11e8-92cd-8ccc848ee670.png)

Reply:
Great! Yeah, I was wrong with `[0, 4]` - this looks awesome.

Reply:
latest attempt: https://github.com/plotly/plotly.js/pull/2641/commits/3d26d475209d744b889ad45920dce4cf359b1529